### PR TITLE
AT_300.030 | Browse Languages | Language Java | Verify amount of comments in object-oriented version

### PR DESCRIPTION
**Pull request by GalaKon**

### CODE REVIEW

**Comment by GalaKon** [comment]
[US_300.006 | Browse Languages | Language Java | Verify amount of comments in object-oriented version](https://trello.com/c/LeWPS0f1/291-us300006-browse-languages-language-java-verify-amount-of-comments-in-object-oriented-version)

**Comment by GalaKon** [comment]
[AT_300.006 | Browse Languages | Language Java | Verify amount of comments in object-oriented version](https://trello.com/c/lsff0a8p/292-at300006-browse-languages-language-java-verify-amount-of-comments-in-object-oriented-version)

**Review by VasiliySiniavin — APPROVED** [review]

File: src/test/java/tests/GalaKonTests.java [hunk 1]
```diff
@@ -33,6 +29,25 @@ public void checkTheNameOfMathematicaLanguageAuthor() {
         String actualResult = lmp.BrentonBostick().getText();
 
         Assert.assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void confirmObjectOrientedVersionHasTheLargestNumberOfComments() {
```
Comment (Rom-T) [inline src/test/java/tests/GalaKonTests.java:35]: The name of the test must contain the word "test"

**Comment by GalaKon** [comment]
The changes in project were deleted with accidentally deleted branch, that's why it's needed to create a new pull request with a new branch.

**Pull request closed without merging**